.github/workflows/publish_oci_containers.yml

```diff
@@ -349,3 +349,103 @@ jobs:
             --container ${{ needs.determine_environment.outputs.repository }} \
             --version ${{ inputs.version }} \
             --manifest_folder manifests
+
```

Comment:
@NotTheEvilOne Can `gl-oci` do these `podman` operations already?

Reply:
I mostly just mirrored the way [.github/workflows/publish_kmodbuild_container.yml](https://github.com/gardenlinux/gardenlinux/blob/3077997cb5a081353ef0d3480f111729aa8aafb6/.github/workflows/publish_kmodbuild_container.yml) works. If `gl-oci` can do this, then we should probably do a separate PR after this to migrate both to it.

But unless this is a really quick change that we can be sure works, I'd prefer not stalling this PR till then.